terraform-provider-constellation/internal/provider/shared_attributes.go

```diff
@@ -166,6 +170,9 @@ func newImageAttributeSchema(t attributeType) schema.Attribute {
 				MarkdownDescription: "Semantic version of the image.",
 				Computed:            !isInput,
 				Required:            isInput,
+				Validators: []validator.String{
+					stringvalidator.RegexMatches(semverRegex, "must be a valid semantic version"),
```

Comment:
```suggestion
					stringvalidator.RegexMatches(semverRegex, "must be a valid semantic version with a v prefix in the form of vX.Y.Z"),
```
nit: "real" semantic versioning doesnt have the v prefix